src/domains/dom_runner.c

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2023 EPAM Systems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <domain.h>
+#include <xen_dom_mgmt.h>
+#include <zephyr/logging/log.h>
+
+#ifdef CONFIG_AOS_ENABLE_DOMU
+#include "domu/domu_runner.h"
+#endif
+
+LOG_MODULE_REGISTER(domains);
+
+extern struct xen_domain_cfg domd_cfg;
+
+int create_domains(void)
+{
+    int rc;
+
+    rc = domain_create(&domd_cfg, 1);
```

Comment:
Sure, some backends in generic case may be located e.g. in Dom0, but since it is a product with some static configuration without such cases, I prefer to left it as it was.